[2025-02-17T15:31:24.124+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ml_pipeline.load manual__2025-02-17T15:31:15.905209+00:00 [queued]>
[2025-02-17T15:31:24.134+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ml_pipeline.load manual__2025-02-17T15:31:15.905209+00:00 [queued]>
[2025-02-17T15:31:24.135+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T15:31:24.135+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-02-17T15:31:24.136+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T15:31:24.150+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): load> on 2025-02-17 15:31:15.905209+00:00
[2025-02-17T15:31:24.158+0000] {standard_task_runner.py:52} INFO - Started process 277 to run task
[2025-02-17T15:31:24.159+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'load', 'manual__2025-02-17T15:31:15.905209+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/DAG-fraud.py', '--cfg-path', '/tmp/tmp68540_jh', '--error-file', '/tmp/tmpot8b_d2d']
[2025-02-17T15:31:24.161+0000] {standard_task_runner.py:80} INFO - Job 11: Subtask load
[2025-02-17T15:31:24.215+0000] {task_command.py:371} INFO - Running <TaskInstance: ml_pipeline.load manual__2025-02-17T15:31:15.905209+00:00 [running]> on host 8538735272b1
[2025-02-17T15:31:24.286+0000] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=group1
AIRFLOW_CTX_DAG_ID=ml_pipeline
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2025-02-17T15:31:15.905209+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-17T15:31:15.905209+00:00
[2025-02-17T15:31:24.516+0000] {base.py:68} INFO - Using connection ID 'postgres_***' for task execution.
[2025-02-17T15:31:29.713+0000] {logging_mixin.py:115} INFO - Data prediksi berhasil dimuat ke PostgreSQL dalam tabel predictions_table
[2025-02-17T15:31:29.716+0000] {python.py:173} INFO - Done. Returned value was: None
[2025-02-17T15:31:29.729+0000] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=load, execution_date=20250217T153115, start_date=20250217T153124, end_date=20250217T153129
[2025-02-17T15:31:29.760+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-02-17T15:31:29.784+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
